CONTRIBUTING

```diff
@@ -53,3 +53,20 @@ guidelines:
     documentation. Please look at the "pod" files in doc/man[1357] for
     examples of our style. Run "make doc-nits" to make sure that your
     documentation changes are clean.
+
+    7.  For user visible changes (API changes, behaviour changes, ...),
+    consider adding a note in CHANGES.  This could be a summarising
+    description of the change, and could explain the grander details.
+    Have a look through existing entries for inspiration. 
+    Please note that this is NOT simply a copy of git-log oneliners.
+    Also note that security fixes get an entry in CHANGES.
```

Comment:
It's meant as a side note, since security fixes usually don't come as public PRs.  It's meant to explain the occurrence of entries with CVE identifiers.  Thing is that pure bug fixes don't usually count as user visible changes, so they wouldn't normally get an entry.  Security fixes are an exception in this regard.

Reply:
Ok. Then you might as well leave the sentence as it is.

Reply:
Okie